### ISSUE TITLE
[Details List]- User is not able to access the 'checkbox' control while navigating using narrator.

### ISSUE THREAD

**Issue by Harish9450** [OfficeDev/office-ui-fabric-react#9466]
### Environment Information 
 
- __Package version(s)__: Office-fabric v6.40.1
- __Browser and OS versions__: Edge & Win-10

### Please provide a reproduction of the bug in a codepen:
https://codepen.io/v-harc/pen/RzWdjz
 
#### Actual behavior:
User is not able to access the 'checkbox' control while navigating using narrator. When narrator focus is on '`Item 0` Check box and press enter/space key, checkbox does not get selected'.
 
#### Expected behavior:
User should be able to access the 'checkbox' control while navigating using narrator. When narrator focus is on '`Item 0` Check box and press enter/space key, checkbox should get select'.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? ( No)
 
Requested priority: (Normal) 
 
Products/sites affected: (if applicable)

**Comment by natalieethell** [comment]
Hey @Harish9450 thank you for opening an issue and providing a CodePen. I'm not currently able to reproduce the behavior you're seeing, but I may be missing something. 

In going through your CodePen in Scan Mode, I'm able to navigate through each cell of the DetailsRow using arrow keys and select the row when focused on the checkbox using space/enter. With Scan Mode off, I'm able to navigate through each row and select the row using space/enter. Am I missing something? Were you experiencing issues in Scan Mode?

**Comment by Harish9450** [comment]
Repro Steps-
1- Open code pan
2- Go to the first row by tab key
3- Enable scan mode
4- Press Enter/Space key and observe the problem (i.e Checkbox is not getting checked/unchecked by pressing Enter/Space key in scan mode).

![bug](https://user-images.githubusercontent.com/38786340/59672059-6da6ee80-91dc-11e9-9d79-ad1b7a3810d6.png)

**Comment by natalieethell** [comment]
@Harish9450 thanks for the clarification! I see your issue. We'll take a look.

**Comment by natalieethell** [comment]
@Harish9450 so we're going to check with the Narrator/Edge team to see if they have an idea as to what's going on here. We'll report back once we get an answer from them!

**Comment by natalieethell** [comment]
We're still waiting to hear back from the Edge team - sorry about the delay.

**Comment by xugao** [comment]
this bug is scan mode only. and most likely an external/narrator issue, @natalieethell - could you link the issue you reported?
downgrade the bug to p3

**Comment by cschlechty** [comment]
When changing the scan mode, focuses isn't on the checkbox but rather outer row. It looks like the event handler doesn't fire there, but the user can scan down to the checkbox itself.

**Comment by cschlechty** [comment]
The user can also scan down to the next cell which contains the checkbox to perform selection. It's reasonable to assume a user would want to interact directly with the control in scan mode.
